08_deploy_bmo.sh

```diff
@@ -5,12 +5,16 @@ eval "$(go env)"
 # Get the latest bits for baremetal-operator
 export BMOPATH="$GOPATH/src/github.com/metalkube/baremetal-operator"
 
-oc --as system:admin --config ocp/auth/kubeconfig new-project bmo-project
+# Make a local copy of the baremetal-operator code to make changes
+cp -r $BMOPATH .
```

Comment:
We really just need the deployment files, right? We don't need to copy the whole git repository? Should we put them in the ocp directory, since that's where we are putting all of our other temporary files like the master ignition files?

Reply:
Ok. I was working on creating a deploy directory under dev-scripts instead of copying the whole repo. But, I can put it into the ocp directory too.